voting-contentApp/apps/voting-app.yaml

```diff
@@ -0,0 +1,13 @@
+!content-type:poll
```

Comment:
This is an old definition style, should follow with latest documentation. BTW, this one should be go after FA-2 vs 4

Reply:
I have rechecked the latest document, but it tells the same thing as I defined. Document reference here:
https://docs.magnolia-cms.com/product-docs/6.2/Apps/My-first-content-app/Part-I---Creating-a-basic-content-app.html#_customizing_the_app_descriptor